src/aml_type/aml.address.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const constants = require('../yoti_common/constants');
+const POSTCODE_ATTR = constants.POSTCODE_ATTR;
+const COUNTRY_ATTR = constants.COUNTRY_ATTR;
+
+exports.AmlAddress = class AmlAddress {
+
+  constructor (countryCode, postcode = '') {
+    this.countryCode = countryCode;
+    this.postcode = postcode;
+
+    this.checkCountryCode();
+  }
+
+  /**
+   * @param countryCode
+   */
+  setCountryCode (countryCode) {
+    this.countryCode = countryCode;
+  }
+
+  /**
+   * @returns country
+   */
+  getCountryCode () {
+    return this.countryCode;
+  }
+
+  /**
+   * @param postcode
+   */
+  setPostcode (postcode) {
+    this.postcode = postcode;
+  }
+
+  /**
+   * @returns postcode
+   */
+  getPostcode () {
+    return this.postcode;
+  }
+
+  /**
+   * @returns {{}}
+   */
+  getData () {
+    let data = {};
+    data[POSTCODE_ATTR] = this.postcode;
+    data[COUNTRY_ATTR] = this.countryCode;
+
+    return data;
+  }
+
+  checkCountryCode () {
+    if(this.countryCode === '') {
+      throw new Error('CountryCode cannot be empty');
```

Comment:
Is it also worth checking that the country code doesn't have 3 letters?

Reply:
imo it's better to keep this check in one place, `Connect API` is already checking this an returning the error. It sounds like duplicate to me. What is Connect API changes the error message or starts accepting 2 letters, we will have to change it here as well.

Reply:
AML check do requires `pem` file as it is calling `signedRequest` module to sign the message and using `request` module to send the request.
Please give your opinion @qzagarese, @laurent-yoti

Reply:
Also `request` and `getRSASignatureForMessage` module shouldn't be tight to some configuration variables. If these were to move into a `Yoti` package that will be shared across the company. It will be impossible to achieve this scenario. And right now it's shared by `ActivityDetails` and `AML check`. 
These are just reusable dependencies/modules that shouldn't rely on any other element to function.

Reply:
Please add basic testing and maybe we can get another general review from @laurent-yoti 
Also, make sure you update from master.